### DISCUSSION TITLE
skip a numerical habit

### DISCUSSION THREAD

**Discussion by ke352802081770314** [iSoron/uhabits#972] [Feature Requests]
as far as I can see, skipping isn't possible for numerical habits. For example, I have gardening, with a goal of 30 minutes daily. However if the weather is bad all day I am prevented from doing it, so I would like to be able to mark that day as a skip.
Or did I just not find the way to do it? Thanks! ( I marked this feature request on the assumption it isn't possible yet.)

Edit: Meanwhile the thought occurred to me that it would be even better if I was able to enter a number and still mark it as a skip day, if I was prevented from reaching my goal by circumstances, and I want to record what I was able to achieve, while not letting it count as a fail.

**Comment by hiqua** [comment]
> Meanwhile the thought occurred to me that it would be even better if I was able to enter a number and still mark it as a skip day, if I was prevented from reaching my goal by circumstances, and I want to record what I was able to achieve, while not letting it count as a fail.

In this case the value would be remembered but would not count toward any score or goal, right? Or did you have something else in mind?

**Reply by ke352802081770314** [reply]
yes, that's what I meant. Since it is a skipped day, the value does not influence any scores, but can be seen for information in the timeline. I noticed that reaching a numeric goal seems to be represented by color, the color of the goal for yes or numeric x>=goal, dark gray for no or 1<=x<goal and light grey for ? or 0, and the skip dash is colored like the yes tick mark. So it might be visualized by the goal color despite being too small but probably that is too subtle to be the only indication. also use italic maybe? or show it in parentheses?
